controllers/tag.js

```diff
@@ -17,4 +17,47 @@ router.post('/', async (req, res) => {
   }
 });
 
+
+
+// @route:get "/tag" || "/tag?isSchool=xyz"
+// @desc:Get all tags if no query
+// get only school tags if url: "/tags?isSchool=true"
+router.get("/tag", async (req, res) => {
+  try {
+      const isSchool = req.query.isSchool
```

Comment:
This implementation is pretty specific, only for one attribute of the schema. Try to make it more general. This is more like hardcoding.